### PR TITLE
Fix incorrect steps in the documentation

### PR DESCRIPTION
**Pull request by calvinaco**
- Fix the git clone command as `CosmWasm/cosmwasm-examples` has updated its `master` branch directory structure
- Force `cosmwasm` to output as JSON in one of the commands for extracting information

### CODE REVIEW

File: getting-started/compile-contract.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ to a pre-defined beneficiary. First, clone the repo and try to build the wasm bu
 ```shell
 # get the code
 git clone https://github.com/CosmWasm/cosmwasm-examples
-cd cosmwasm-examples/escrow
 git checkout escrow-0.7.0
+cd cosmwasm-examples/escrow
```
Comment (alpe) [inline getting-started/compile-contract.md:23]: This does not work. Steps should be
```sh
cd cosmwasm-examples
git checkout escrow-0.7.0
cd escrow
```

File: getting-started/interact-with-contract.md [hunk 2]
```diff
@@ -43,8 +43,8 @@ wasmd tx wasm instantiate $CODE_ID "$INIT" \
     --from fred --amount=50000umayo  --label "escrow 1" $TXFLAG -y
 
 # check the contract state (and account balance)
-wasmd query wasm list-contract-by-code $CODE_ID $NODE
-CONTRACT=$(wasmd query wasm list-contract-by-code $CODE_ID $NODE | jq -r '.[0].address')
+wasmd query wasm list-contract-by-code $CODE_ID $NODE --output=json
+CONTRACT=$(wasmd query wasm list-contract-by-code $CODE_ID $NODE --output=json | jq -r '.[0].address')
```
Comment (alpe) [inline getting-started/interact-with-contract.md:47]: 👍  perfect. I can imagine that non json output created some trouble

**Review by alpe — CHANGES_REQUESTED** [review]
Good 👀  . Thanks for your contribution!
I did a clean checkout to test the `cd path`. Can you verify and update the PR, please?

**Comment by orkunkl** [comment]
Applied the changes to 0.14. The branching got quite messy so had to insert the change manually, thank you

**Pull request closed without merging**